com/win32com/client/CLSIDToClass.py

```diff
@@ -17,7 +17,9 @@
 Access.
 """
 
-mapCLSIDToClass = {}
+from __future__ import annotations
+
+mapCLSIDToClass: dict[str, type] = {}
```

Comment:
Not necessary for this change. It just helped be make sense of, and validate the type narrowing.